packages/types/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "types",
```

We should adhere to our package naming convention within the repository

```suggestion
  "name": "@osmosis-labs/types",
```